mta-v7.x/mta.sw.yaml

```diff
@@ -359,7 +420,7 @@ states:
           refName: createNotification
           arguments:
             title: "MTA analysis completed"
-            message: '"MTA analysis is completed and the report for " + .application.repository.url + " is ready."'
+            message: '"MTA analysis is completed with " + (.application.nonMandatoriesIncidents|tostring) + " non mandatories incidents and the report for " + .application.repository.url + " is ready."'
```

Comment:
s/non mandatories../non-mandatory incidents. The report for " + .application.repository.url + " is ready."/